board/stm32fx/peripherals.h

```diff
@@ -5,6 +5,15 @@ void gpio_usb_init(void) {
   GPIOA->OSPEEDR = GPIO_OSPEEDER_OSPEEDR11 | GPIO_OSPEEDER_OSPEEDR12;
 }
 
+void gpio_spi_init(void) {
```

Comment:
This might be more board-specific, why not just have this in `dos` and `red_chiplet`?

Reply:
because we don’t do the whole board init in the flasher, but perhaps that’s a legacy thing?

Reply:
Ah, I see.
Should be fine to keep it as-is then